docs/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -3,12 +3,12 @@
 :::info Required roles: system administrator, security administrator
 :::
 
-The OpenID/Connect ([OIDC](https://openid.net/specs/openid-connect-core-1_0.html)) protocol adds an authentication layer on top of the [OAuth2](https://www.rfc-editor.org/rfc/rfc6749) Authorization protocol.
+The OpenID Connect ([OIDC](https://openid.net/specs/openid-connect-core-1_0.html)) protocol adds an identity layer on top of the [OAuth2](https://www.rfc-editor.org/rfc/rfc6749) Authorization protocol.
 
 OIDC authentication, together with the z/OS [Identity Propagation](https://www.redbooks.ibm.com/redbooks/pdfs/sg247850.pdf) mechanism, is the foundation of the API ML Identity Federation.
 In this article, OIDC is often referred to as the provider, while the token-related functionality is actually provided by the OAuth2 component of the OIDC implementation.
 
-You can configure Zowe API ML to authenticate users by accepting Access Tokens issued by an external OIDC/OAuth2 provider.
+You can configure Zowe API ML to authenticate users by accepting Access Tokens issued by an external OIDC provider.
 This configuration is useful in advanced deployments of Zowe where client applications need to access mainframe as well as enterprise/distributed systems while simultaneously offering single sign-on (SSO) across system boundaries.  
 
 This article details the API ML OIDC authentication functionality, and how to configure the OIDC Authentication feature.
```

Comment:
3 lines below. Is it still valid that ACF2 is not supported, when we have the internal mapper?